### PR TITLE
Add Github labels for git.k8s.io/k8s.io

### PR DESCRIPTION
**Pull request by ameukam**
Part of https://github.com/kubernetes/k8s.io/issues/508.

cc @munnerz 

/wg k8s-infra

### CODE REVIEW

**Comment by munnerz** [comment]
Thanks for taking a look into this 😀 these seems like a sensible set of labels, and we can expand/alter in future if needs be 🙂

/lgtm

File: label_sync/labels.yaml [hunk 1]
```diff
@@ -808,6 +808,33 @@ repos:
...
+        addedBy: anyone
+      - color: 0052cc
+        description: Issues or PRs related to monitoring of the Kubernetes project infrastructure
+        name: area/infra/monitoring
+        target: both
+        prowPlugin: label
+        addedBy: anyone
+      - color: 0052cc
+        description: Issues or PR related to reliability of the Kubernetes project infrastructure
+        name: area/infra/reliability
```
Comment (cblecker) [inline label_sync/labels.yaml]: missing `target: both`

File: label_sync/labels.yaml [hunk 2]
```diff
@@ -808,6 +808,33 @@ repos:
...
+        prowPlugin: label
+        addedBy: anyone
+      - color: 0052cc
+        description: Issues or PRs related to Publishing bot
+        name: area/infra/publishing-bot
```
Comment (cblecker) [inline label_sync/labels.yaml]: missing `target: both`

File: label_sync/labels.yaml [hunk 3]
```diff
@@ -808,6 +808,33 @@ repos:
...
+        prowPlugin: label
+        addedBy: anyone
+      - color: 0052cc
+        description: Issues or PRs related to Cert Manager
+        name: area/infra/cert-manager
```
Comment (cblecker) [inline label_sync/labels.yaml]: missing `target: both`

File: label_sync/labels.yaml (hunk 1 above)
Comment (ameukam) [reply label_sync/labels.yaml]: Added.

File: label_sync/labels.yaml (hunk 2 above)
Comment (ameukam) [reply label_sync/labels.yaml]: Added

File: label_sync/labels.yaml (hunk 3 above)
Comment (ameukam) [reply label_sync/labels.yaml]: Added

**Pull request merged**